packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -321,11 +323,21 @@ class ReactSixteenThreeAdapter extends EnzymeAdapter {
     let isDOM = false;
     let cachedNode = null;
     return {
-      render(el, context) {
+      render(el, context, {
+        providerValues = new Map(),
+      } = {}) {
```

Comment:
I'd like to make the third argument be an object. This way we can easily pass additional params as enzyme/react evolves without introducing breaking changes.